release-notes-5.0.0.md

```diff
@@ -14,5 +12,11 @@ ${version-number}
   - If the task has been paused, and the `CAF_WORKER_PAUSED_QUEUE` environment variable is NOT set, the worker process the task as
   normal (as if the task was not paused).
 
+- SCMOD-12319: Removed worker-document archetype
+    The following archetypes have been removed:  
+      - Worker Framework Archetype  
+      - Batch Worker Framework Archetype  
```

Comment:
Removal of the Worker Framework archetype and the Batch Worker Framework archetype have nothing to do with this repository.